frontend/app/(marketing)/page.tsx

```diff
@@ -53,7 +53,7 @@ export default async function HomePage() {
         </div>
       </nav>
 
-      <main className="min-h-screen bg-white pt-20">
+      <main className="bg-background min-h-screen pt-20">
```

Comment:
Let's keep bg-white here instead, as we want this section to be white in both modes.
```suggestion
      <main className="bg-white min-h-screen pt-20">
```